WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -103,6 +111,7 @@
         ReaderWebViewUrlClickListener,
         BasicFragmentDialog.BasicDialogPositiveClickInterface {
     private static final String BOOKMARKS_SAVED_LOCALLY_DIALOG = "bookmarks_saved_locally_dialog";
+    private static final int FILE_DOWNLOAD_PERMISSION_REQUEST_CODE = 123;
```

Comment:
Not fully sure if it can make sense but I would consider to add this request code in the list [here](https://github.com/wordpress-mobile/WordPress-Android/blob/0b681e358496663bce0ed4a4eb133c2738fdbe37/WordPress/src/main/java/org/wordpress/android/util/WPPermissionUtils.java#L34). wdyt?

Reply:
good point